updated: new KMM version (#4249)
JetBrains/kotlin-web-site: docs/topics/multiplatform/multiplatform-plugin-releases.md

**0.8.2**

Released: 16 May, 2024

</td>
<td>

> * Support for Android Studio Jellyfish as well as for the new Canary version, Koala.

Reviewer: ```suggestion
* Support for Android Studio Jellyfish and for the new Canary version, Koala.
```
